main.py

```diff
@@ -1,12 +1,251 @@
+        ## "8": "Delete 1 Customer "    
+        elif choice == '8':
+            customer.delete_customer()
+            print()
+            print("The Customer has been deleted.")
+            print()
+            print(customer.list_customers())
+        ## "9": "View  Info on 1 Customer"    
+        elif choice == '9':
+            select_by = input("Would you like to select by? Enter name or id: \n ")
+            if select_by=="name":
+                name = input("Which customer name would you like to locate? \n")
+                customer.get_customer(name=name)
+            elif select_by=="id":
+                id = input("Which customer id would you like to locate? \n")
+                if id.isnumeric():
+                    id = int(id)
+                    customer.get_customer(id=id)
+            else:
+                print("Could not locate. Please enter a valid customer id or name")
+            
+            if customer.selected_customer:
+                print()
+                print("Selected Customer: ", customer.selected_customer)
+        ## "10": "View all Customers"    
+        elif choice == '10':
+            for customer in customer.list_customers():
+                print(customer)
+        # "11" : "Check in 1 Video"    
+        elif choice == '11':
+            print("Let's get started with the check-in process!\n ")
+            video_id = input("What is the video id: ")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                video.selected_video = video.get_video(id=video_id)
+                print(video.selected_video) 
+            customer_id = input("What is the customer's id checking ing the video: ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                customer.selected_customer = customer.get_customer(id=customer_id)
+                print(customer.selected_customer)
+            
+            checked_in_rental = rental.check_in(customer_id, video_id)
+            print(checked_in_rental)
+            print(" Success ! The video was checked in!")
+
+        ## "12": "Check out 1 Video Return"   
+        elif choice == '12':
+            print("Let's get started with the check-out process!\n ")
+            video_id = input("What is the video id: ")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                video.selected_video = video.get_video(id=video_id)
+                print(video.selected_video)
+
+            customer_id = input("What is the customer's id: ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                customer.selected_customer = customer.get_customer(id=customer_id)
+                print(customer.selected_customer)
+            
+            checked_out_rental = rental.check_out(customer_id, video_id)
+            print("Success! The video was checked out!")
+            print(checked_out_rental)
```

Comment:
Just as with the check-in, consider how your CLI should handle the situation where the check-out cannot be completed (for instance, due to not available inventory).